### PR TITLE
Add room with dummy migration

### LINKED ISSUE #4: Check compatibility of Room with existing db.
Migration should be smooth.

### PR DESCRIPTION
**Pull request by Apsaliya**
closes #4

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -4,10 +4,19 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
+apply plugin: 'kotlin-kapt'
+
 android {
+  signingConfigs {
```
Comment (Apsaliya) [inline app/build.gradle]: This needs to be removed. Will do separately.

File: app/src/main/java/wordtextcounter/details/main/store/entities/Report.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(tableName = "Details")
+data class Report(
+    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") var id: Int?,
+    @ColumnInfo(name = "name") var name: String?,
+    @ColumnInfo(name = "data_text") var dateText: String?,
+    @ColumnInfo(name = "word") var words: String?,
```
Comment (Hirak20) [inline app/src/main/java/wordtextcounter/details/main/store/entities/Report.kt]: Word char etc can be stored as an to Int

**Comment by Apsaliya** [comment]
I had git war locally So can't change this in this one..Will address changes in separate PR.

File: app/src/main/java/wordtextcounter/details/main/store/entities/Report.kt (hunk 2 above)
Comment (Apsaliya) [reply app/src/main/java/wordtextcounter/details/main/store/entities/Report.kt]: Makes sense. Can do.

**Pull request merged**